spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -339,7 +339,7 @@ public void destroy() {
 
 		groupedMessages.forEach((ProjectSubscriptionName psName, List<String> ackIds) -> {
 
-			ApiFuture<Empty> ackApiFuture = asyncOperation.apply(psName.getSubscription(), ackIds);
```

Comment:
This used to work before #1880 because there was another bug that was cancelling this one out.
We need to pass the fully-qualified subscription name, which you get by calling `toString()` on `ProjectSubscriptionName`.